oxidation/libparsec/crates/core/src/error.rs

```diff
@@ -74,3 +74,13 @@ impl From<TrustchainError> for RemoteDevicesManagerError {
         Self::InvalidTrustchain { exc }
     }
 }
+
+#[derive(Error, Debug, PartialEq, Eq)]
+pub enum LoggedCoreError {
+    #[error("The index provided is invalid: {index}")]
```

Comment:
Rename `index` by `handle`: while implemented as an index, this is not relevant for the user.

"Handle" on the other hand designed an opaque id that is provided to track a resource.

Reply:
Done